### PR TITLE
Docu core

### PR DESCRIPTION
**Pull request by ornemarioni**

### CODE REVIEW

File: galaxychop/core.py [hunk 1]
```diff
@@ -894,12 +915,22 @@ class Columns(enum.Enum):
     """Columns name used to decompose galaxies."""
 
     m = 0
+    """Masses"""
```
Comment (antotaverna) [inline galaxychop/core.py:918]: (m, x, y, z, vx, vy, vz ) are stellar particles. I think. @ValeCristiani ?

Comment (vcristiani) [reply galaxychop/core.py:918]: Yes, are stellar particles.

File: galaxychop/core.py [hunk 2]
```diff
@@ -772,26 +754,41 @@ def jcirc(self, bin0=0.05, bin1=0.005):
...
         Notes
         -----
         J_z : z-component of normalized specific angular momentum.
-        J_circ : circular specific angular momentum.
-        J_p : module of the projection on the xy plane of the normalized
-        specific angular momentum.
+
+        J_circ : Specific circular angular momentum.
+
+        J_p : Absolute value of the projection on the xy plane of the
```
Comment (vcristiani) [inline galaxychop/core.py:779]: Again it is not an absolute value, but the quantity is positive by definition.

File: galaxychop/core.py [hunk 3]
```diff
@@ -849,4 +846,91 @@ def paramcirc(self):
...
+
+        Returns
+        -------
+        X : `np.ndarray(n,7)`
+            2D array where each file it is a diferent particle and
+            each column is a parameter of the particles (E_star, eps, eps_r)
+        y : `np.ndarray(n)`
+            1D array where is identified the nature of each particle
+            0=star, 1=gas and 2=dark matter
+        """
```
Comment (vcristiani) [inline galaxychop/core.py:880]: If you look at what this method does, it does not correspond to the documentation provided.

File: docs/source/tutorial.ipynb [hunk 4]
```diff
@@ -282,8 +256,8 @@
    "source": [
     "This returns a new galaxy with the parameters:\n",
     "\n",
-    "* **x**: Stellar normalized energy.\n",
-    "* **y**: z component of the normalized angular momentum.\n",
+    "* **x**: Normalized specific energy for the particle with the maximum z-component of the normalized specific energy per bin.\n",
+    "* **y**: Maximum value of the z-component of the normalized specific energy per bin.\n",
```
Comment (vcristiani) [inline docs/source/tutorial.ipynb:260]: **z-component of normalized specific angular momentum**

**Review by vcristiani — CHANGES_REQUESTED** [review]
The documentation is generally good. There are some changes to be made in the description of some variables, and revisit the value method documentation because it does not match what the method does.

**Review by vcristiani — APPROVED** [review]
Everything looks good

**Review by RafaelPignata622 — APPROVED** [review]

**Review by antotaverna — APPROVED** [review]

**Pull request merged**